src/main/resources/db/migration/V1_1101__SPLASH_SCREEN_FUNCS.sql

```diff
@@ -0,0 +1,18 @@
+create table SPLASH_SCREEN_FUNCS
```

Comment:
This table is not used in Nomis but the table I am interested in depends on it.
The table allows pl/sql functions to be added - but currently this is not done